src/components/PendingSwapModal.tsx

```diff
@@ -198,22 +198,26 @@ const PendingSwapModal = ({
   )
 
   return (
-    <>
+    <DialogContent>
       {(currentStep === "timer" || currentStep === "exchange") && (
-        <div className={styles.virtualSwapModal}>
-          <div className={styles.headerContent}>
-            <b className={styles.title}>
+        <div>
+          <div>
+            <b>
               {formattedSynthBalance} {synthTokenFrom.symbol} {"->"}{" "}
               {tokenTo.symbol}
             </b>
           </div>
-          <div className={styles.centerContent}>
+          <div>
             {currentStep === "timer" && (
-              <div className={styles.timer}>
-                <h2>
-                  {minutesRemaining} {t("minRemaining")}
-                </h2>
-              </div>
+              <Typography
+                variant="h2"
+                color="primary"
+                textAlign="center"
+                mt={4}
+              >
+                {t("waiting")}
+                {minutesRemaining} {t("minRemaining")}
```

Comment:
This isn't your bug, but please us `{t("minutesLeft", { count: minutesRemaining })}` to correctly pluralize the translation